.angular-cli.json

```diff
@@ -19,9 +19,16 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "styles": [
-        "styles.scss",
-        "assets/scss/main.scss"
+        "assets/scss/styles.scss"
```

Comment:
I don't think we should remove the reference to styles.scss, because that what the Angular CLI starts with, and odds are, a lot of folks will be on projects where they won't have the authority to change to the assets-folder style. 

Having both in this in this array, though, is a great idea because:

* It shows how you can customize the `.angular-cli.json` file
* It does introduce them to the Bermon SASS system

Reply:
@vjwilson I sort of agree with you there, but if we leave them with both options it could potentially cause some confusion + you would have to offer the justification for doing so, the pros/cons, and lead to a debate.

Reply:
@vjwilson @bellizio I sorta agree too. What I tend to do is create a main.scss file for the assets folder. That's what I did with my PR.

Reply:
Ok I'll leave both in there, but change assets/styles.scss to main.scss to avoid potential confusion